### DISCUSSION TITLE
Track score seekbar header option

### DISCUSSION THREAD

**Discussion by aaprasad** [talmolab/sleap#1804] [Ideas]
Currently it is not possible to view the track score of an instance in the SLEAP gui. It would be nice to A. have the track score display in the instance section and change the "score" to be specific to `instance score` and B. in the seekbar header have a heuristic based on the track score be computed, maybe the difference between the top two scores for that instance or something.

**Comment by roomrys** [comment]
Long awaited, but finally integrated. This issue has been fixed in the SLEAP 1.4.1 release!
